### PR TITLE
chore(*): Applying java codestyles

### PR DESCRIPTION
**Pull request by robzienert**

### CODE REVIEW

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/admin/AdminController.java [hunk 1]
```diff
@@ -15,63 +15,67 @@
...
+        pollingMonitors.stream()
             .filter(it -> it.getName().equals(monitorName))
             .findFirst()
-            .orElseThrow(() -> new NotFoundException(format(
-                "PollingMonitor %s was not found, available monitors are: %s",
-                monitorName,
-                pollingMonitors.stream().map(PollingMonitor::getName).collect(Collectors.toList())
-            )));
+            .orElseThrow(
+                () ->
```
Comment (emjburns) [inline igor-web/src/main/groovy/com/netflix/spinnaker/igor/admin/AdminController.java]: this section is not as nice as it was before, I will admit.

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitLabConfig.java [hunk 2]
```diff
@@ -30,40 +31,40 @@
...
-            .setEndpoint(Endpoints.newFixedEndpoint(address))
-            .setRequestInterceptor(new PrivateTokenRequestInterceptor(privateToken))
-            .setClient(new OkClient())
-            .setConverter(new JacksonConverter())
-            .build()
-            .create(GitLabClient.class);
-    }
+  public GitLabClient gitLabClient(String address, String privateToken) {
+    return new RestAdapter.Builder()
+        .setEndpoint(Endpoints.newFixedEndpoint(address))
```
Comment (emjburns) [inline igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitLabConfig.java]: spacing looks a bit weird here, no?

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitLabProperties.java [hunk 3]
```diff
@@ -16,48 +16,42 @@
...
 @ConditionalOnProperty("gitlab.base-url")
 @ConfigurationProperties(prefix = "gitlab")
 public class GitLabProperties {
-    @NotEmpty
-    private String baseUrl;
+  @NotEmpty private String baseUrl;
 
-    @NotEmpty
-    private String privateToken;
+  @NotEmpty private String privateToken;
```
Comment (emjburns) [inline igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitLabProperties.java]: Are we putting all annotations on the same line? or just this one?

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/admin/AdminController.java (hunk 1 above)
Comment (robzienert) [reply igor-web/src/main/groovy/com/netflix/spinnaker/igor/admin/AdminController.java]: Too bad!

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitLabConfig.java (hunk 2 above)
Comment (robzienert) [reply igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitLabConfig.java]: Too bad!

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitlabCiConfig.java [hunk 4]
```diff
@@ -34,78 +38,82 @@
...
-        log.info("creating gitlabCiMasters");
-        Map<String, GitlabCiService> gitlabCiMasters = gitlabCiProperties.getMasters().stream()
-            .map(gitlabCiHost ->
-                gitlabCiService(igorConfigurationProperties, "gitlab-ci-" + gitlabCiHost.getName(), gitlabCiHost, objectMapper))
+  @Bean
+  public Map<String, GitlabCiService> gitlabCiMasters(
+      BuildServices buildServices,
+      final IgorConfigurationProperties igorConfigurationProperties,
+      GitlabCiProperties gitlabCiProperties,
+      ObjectMapper objectMapper) {
```
Comment (emjburns) [inline igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitlabCiConfig.java]: oo this is weird formatting. should do either adam or fletch style, not both

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitLabProperties.java (hunk 3 above)
Comment (robzienert) [reply igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitLabProperties.java]: It's the way Google's code format forces... so... yes?

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitlabCiConfig.java (hunk 4 above)
Comment (robzienert) [reply igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitlabCiConfig.java]: Too bad!

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitlabCiConfig.java [hunk 5]
```diff
@@ -34,78 +38,82 @@
...
+        host.getLimitByMembership(),
+        host.getLimitByOwnership(),
+        host.getPermissions().build());
+  }
 
-    public static GitlabCiClient gitlabCiClient(String address, String privateToken, int timeout, ObjectMapper objectMapper) {
-        OkHttpClient client = new OkHttpClient();
-        client.setReadTimeout(timeout, TimeUnit.MILLISECONDS);
+  public static GitlabCiClient gitlabCiClient(
+      String address, String privateToken, int timeout, ObjectMapper objectMapper) {
```
Comment (emjburns) [inline igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitlabCiConfig.java]: again weird formatting here...

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/LockManagerConfig.java [hunk 6]
```diff
@@ -21,51 +21,50 @@
...
-import java.time.Duration;
-import java.util.Optional;
-
 @Configuration
 @ConditionalOnProperty("locking.enabled")
 @EnableConfigurationProperties(LockManagerConfigProperties.class)
 public class LockManagerConfig {
-    @Autowired
-    LockManagerConfigProperties lockManagerConfigProperties;
+  @Autowired LockManagerConfigProperties lockManagerConfigProperties;
```
Comment (emjburns) [inline igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/LockManagerConfig.java]: annotation weirdness

**Pull request merged**